README.md

```diff
@@ -30,6 +30,7 @@ django-admin startproject theprojectname --extension py,yml,json --name Procfile
 ```
 - [ ] Above: don't forget the `--extension` and `--name` params! (also don't forget to change `theprojectname` to your project's name).
 - [ ] Navigate to the project's directory through your command line.
+- [ ] Adding a dot on the command between projectname and --extension would extract on same folder, and not a level below.
```

Comment:
How do you feel about adding the example to this line with the dot included? @elfelround